### PR TITLE
v0.8.0

### LINKED ISSUE #20: update landing page with gnark specific descriptions
#19 need a bit of polishing to update the landing page with gnark specific text (right now we the quick links section looks like for EthSigner;

image

). 

Also few broken links for example: 
https://docs.gnark.consensys.net/Reference/api -_> "Front End" and "Back End" links are broken ( https://pkg.go.dev/github.com/consensys/gnark@%7B%7Bcontent_vars.gnark_version%7D%7D/frontend )

### PR DESCRIPTION
**Pull request by gbotrel**
Update the doc for v0.8.0.

Update benchmark section with [zk](https://zka.lc) and ZPrize links. 

Add section for Profiling circuits. 

Closes #20 #21 #22 . 

## Before creating the PR

Make sure that:

- [ X] You've read the [contribution guidelines](https://consensys.net/docs/doctools/).
- [X ] You've [previewed your changes locally](https://consensys.net/docs/doctools/en/latest/preview/new-system/#preview-locally).

## Impacted parts

For content changes:

- [X ] Documentation content
- [ ] Documentation page organization

## After creating your PR and tests have finished

Make sure that:

- [ ] You've [fixed any issues](https://consensys.net/docs/doctools/en/latest/contribute/fix-cicd-errors/) raised by the tests.
- [ ] You've [previewed your changes on GitHub Pages](https://consensys.net/docs/doctools/en/latest/preview/new-system/#preview-on-github-pages)

### CODE REVIEW

File: docs/HowTo/compile.md [hunk 1]
```diff
@@ -14,3 +14,5 @@ The `frontend.Compile` method takes a high-level program and translates it to a
 var myCircuit Circuit
 r1cs, err := frontend.Compile(ecc.BN254, r1cs.NewBuilder, &myCircuit)
 ```
+
+:::tip Check out the [gnark Playground](https://play.gnark.io) to compile and visualize the sequence of constraints from a circuit. :::
```
Comment (ivokub) [inline docs/HowTo/compile.md:18]:
```suggestion
:::tip Playground

Check out the [gnark Playground](https://play.gnark.io) to compile and visualize the sequence of constraints from a circuit.

:::
```

**Comment by gbotrel** [comment]
@Ezzahhh I'm not admin of this repo, are you? If so can you remove the check "docs/readthedocs.com:pegasys-gnark — Read the Docs build failed!" from PR against all branches? Thanks 👍

**Comment by Ezzahhh** [comment]
> @Ezzahhh I'm not admin of this repo, are you? If so can you remove the check "docs/readthedocs.com:pegasys-gnark — Read the Docs build failed!" from PR against all branches? Thanks 👍

I've given the gnark team maintain access to this repo. Let me know if you want this to be adjusted. I have removed the circleci webhook and am in the process of disabling readthedocs. This PR might still need to be manually merged since the checks might still stick around

**Review by ivokub — APPROVED** [review]
Seems good!

**Comment by gbotrel** [comment]
> > @Ezzahhh I'm not admin of this repo, are you? If so can you remove the check "docs/readthedocs.com:pegasys-gnark — Read the Docs build failed!" from PR against all branches? Thanks 👍
> 
> I've given the gnark team maintain access to this repo. Let me know if you want this to be adjusted. I have removed the circleci webhook and am in the process of disabling readthedocs. This PR might still need to be manually merged since the checks might still stick around

@Ezzahhh I still see 
image

and while a new "Settings" tab appeared, I can't do much from there, it seems.

**Comment by Ezzahhh** [comment]
@gbotrel 

I have removed them in readthedocs and circleci now. I also adjusted the teams permissions to Admin so it's easier for you guys to manage this repo and merge past requirements like now. I think if you're ready you can just force the merge. New PRs shouldn't have these extra requirements anymore 🤞

**Pull request merged**